tests/js/lib/globalErrorHandler.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var jqUnit = require("node-jqunit");
+
+var gpii = fluid.registerNamespace("gpii");
+
+fluid.defaults("gpii.test.webdriver.globalFailureHandler", {
+    gradeNames: ["fluid.component", "fluid.resolveRootSingle"],
+    singleRootType: "gpii.test.globalFailureHandlerHolder",
+    events: {
+        onError: null
+    }
+});
+
+fluid.registerNamespace("gpii.test.webdriver");
```

Comment:
This is unnecessary since you just issued the defaults above

Reply:
Removed.